src/benchmarklib/benchmark_runner.cpp

```diff
@@ -218,7 +218,7 @@ void BenchmarkRunner::_benchmark_ordered() {
     result.duration = _state.benchmark_duration;
     const auto duration_of_all_runs_ns =
         static_cast<float>(std::chrono::duration_cast<std::chrono::nanoseconds>(_state.benchmark_duration).count());
-    const auto duration_seconds = duration_of_all_runs_ns / 1'000'000'000;
+    const auto duration_seconds = duration_of_all_runs_ns / 1e9f;
```

Comment:
I hate the e notation and it doesn't add readability at all IMO.

Reply:
Sublime doesn't like `1'000'000'000f;`, which breaks the formatting until the next `'`. Alternative would be `static_cast<float>(1'000'000'000)` - is that better?

Reply:
Not really better. But should Sublime's formatting be a reason for us?

Reply:
Hmm, actually, Sublime is right:

```
floating-point-literal:
-> decimal-floating-point-literal
   (hexadecimal-floating-point-literal)

decimal-floating-point-literal:
    fractional-constant exponent-part(opt) floating-point-suffix(opt)
    digit-sequence exponent-part floating-point-suffix(opt)
```
(http://eel.is/c++draft/lex.fcon#nt:decimal-floating-point-literal)

Looks like to use a float literal, the number has to either be fractional or use an exponent. Unsure why the Lexer does not like `digit-sequence exponent-part(opt) floating-point-suffix(opt)`.